### ISSUE TITLE
I am unable to install latest version of Angular CLI.

### ISSUE THREAD

**Issue by anjaikr** [request/request#3320]
I am unable to install latest version of Angular CLI.
I have installed Nodejs on my Windows 10 Pro Laptop
**Nodejs 64 bit version: 12.18.1
npm version: 6.13.6**
When I run npm install -g @angular/cli@latest to install latest version of Angular CLI, it gives me following error warning
npm WARN deprecated request@2.88.2: request has been deprecated, see #3142

Installation stucks with message: postinstall: sill install executeActions
Please help me out on resolving this problem

The link didn't help me much, please show me the steps to resolve this issue.

_Originally posted by @anjaikr in https://github.com/request/request/issues/3142#issuecomment-652506526_

**Comment by AkashKumarSingh11032001** [comment]
I am having same issue. Did you find a solution for this?

**Comment by snylilith** [comment]
Same problem here, any ideas...?

**Comment by AkashKumarSingh11032001** [comment]
Hi @snylilith 
after 3 hours of research, I got to know the partial reason for encountering this error
use this precaution
1- admin cmd
2- stable connection {i prefer mobile data }
3-  install node_modle before installing Angular CLI because some component of angular directly depend upon node

By following this , I was able to overcome that error

**Comment by shilpaanthati6** [comment]
When I try to install angular 9 , i got below error. I got check environment variables and added path,but of no use .Please help me out.

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN rollback Rolling back safe-buffer@5.1.2 failed (this is probably harmless): EPERM: operation not permitted, lstat 'C:\Users\Sindhu\AppData\Roaming\npm\node_modules\@angular\cli\node_modules\string_decoder\node_modules'
npm ERR! code EEXIST
npm ERR! path C:\Users\Sindhu\AppData\Roaming\npm\node_modules\@angular\cli\bin\ng
npm ERR! dest C:\Users\Sindhu\AppData\Roaming\npm\ng
npm ERR! EEXIST: file already exists, cmd shim 'C:\Users\Sindhu\AppData\Roaming\npm\node_modules\@angular\cli\bin\ng' -> 'C:\Users\Sindhu\AppData\Roaming\npm\ng'
npm ERR! File exists: C:\Users\Sindhu\AppData\Roaming\npm\ng
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sindhu\AppData\Roaming\npm-cache\_logs\2020-07-09T17_28_14_402Z-debug.log

**Comment by AkashKumarSingh11032001** [comment]
Hi @shilpaanthati6 this error usually occurs due to two main reason :
1st poor connectivity [ solution mentioned at top ]
2nd npm file crashes
you can follow this -
->delete package-lock.json and node-modules dir
->npm cache clean --force
->npm install
->npm start

**Comment by shilpaanthati6** [comment]
Hi AkashKumarSingh11032001, 

I got cleared this error by removing package-lock.json and node-modules dir.
> 1st poor connectivity [ solution mentioned at top ]
> 2nd npm file crashes
> you can follow this -
> ->delete package-lock.json and node-modules dir
> ->npm cache clean --force
> ->npm install
> ->npm start

Thank you ,
shilpa

**Comment by AlexOkwor** [comment]
I was unable to install the latest version of angular CLI. I followed these steps to fix the error:

First I edited environment variables. 

I changed the path in system variables. By default the path was: C:\Program Files\nodejs\. 

I instructed it to use this path:C:\Users\USER\AppData\Roaming\npm. I simply specified the new path first, so it sees it before the default path example: C:\Users\USER\AppData\Roaming\npm;C:\Program Files\nodejs\;

I uninstalled node.

I reinstalled node using this command: npm --force install -g @angular/cli

It works perfectly now.
